pages/en/lb3/Environment-specific-configuration.md

```diff
@@ -187,6 +187,22 @@ Example data sources:
 }
 ```
 
+You can also configure your datasource._\_env\__.js file to use environment variables:
+
+{% include code-caption.html content="datasources.production.json" %}
+```javascript
+module.exports = {
+  db: {
+    connector: 'mongodb',
+    hostname: process.env.DB_HOST,
+    port: process.env.DB_PORT || 27017,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: 'myapp',
+  }
+```
+
+
```

Comment:
Should I add that blurb about setting `export PRODUCTION=true` or `set PRODUCTION=true` here to make it clearer?